covid_xprize/examples/prescriptors/neat/utils.py

```diff
@@ -87,6 +86,15 @@ def prepare_historical_df():
     return df
 
 
+# Function to load an IPs file, e.g., passed to prescribe.py
+def load_ips_file(path_to_ips_file):
```

Comment:
Could use `load_dataset` from `predictor_scoring.py` to also filter out the geos we're not considering